examples/configure_development.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Configure Auth
+
+# Retrieve the ArgoCD route, api rest server url, and configure the redirect uri.
+ARGOCD_ROUTE=https://$(oc get route argocd-server -n aicoe-argocd-dev -o jsonpath='{.spec.host}')
```

Comment:
remove the `aicoe-argocd-dev` and replace it with `argocd-dev`.
Same with all `aicoe-` prefixed stuff in here.

Later we could even replace it with a configurable placeholder